### ISSUE TITLE
Invalid custom-property values in clr-ui.css

### ISSUE THREAD

**Issue by aravindh-nagarajan** [vmware/clarity#4526]
## Describe the bug

This issue has been raised already (#4441) and part of it has been fixed as part of v3.1.1.

but we have few more properties with the same invalid/undefined values. 

```
--clr-login-background -> has invalid property value
--clr-btn-icon-disabled-color -> uses $clr-btn-default-disabled-color
--clr-forms-select-multiple-error-focus-color -> invalid value
```
![two](https://user-images.githubusercontent.com/49676938/80721714-2d164200-8b1c-11ea-8e0b-dd2f0dbd4b3f.png)
![one](https://user-images.githubusercontent.com/49676938/80721726-2f789c00-8b1c-11ea-80e1-26181898a893.png)
![three](https://user-images.githubusercontent.com/49676938/80721730-30a9c900-8b1c-11ea-9a90-549fef878f0b.png)

## How to reproduce

Sample repository to repro: https://github.com/aravindh-nagarajan/angular-app-without-cli

## Versions

**App**

* Angular: 9.1.0
* Node: 10.15.3
* Clarity: 3.1.1

Please do let me know if you  have any questions?

Thank you.

**Comment by mathisscott** [comment]
@aravindh-nagarajan 
I had some meetings clear up yesterday and was able to get this in review. As before, your screenshots helped tremendously. Thanks for that. I was also able to search through the generated CSS to make sure those were the only SASS variable names that were making it through. They are. I should have thought to do that the first time around. My bad on that. Thanks for your patience.

**Comment by aravindh-nagarajan** [comment]
@mathisscott Thank you for the fast action. 
Have one question. There is a property (`--clr-login-backgroud`).
this property has some encoded string as its value. is it expected or a mistake?
this is also causing our build to fail.
since its not addressed in the PR, so got curious.

**Comment by mathisscott** [comment]
Good catch, @aravindh-nagarajan 

I'll update the PR to fix that

**Comment by mathisscott** [comment]
This has been fixed and backported to v3. It should be released in 3.1.2.
